tests/cross_functional/system_test/test_clone_deletion_after_max_cluster_space_utilization.py

```
@@ -51,7 +50,29 @@ def setup(self, interface_type, pvc_factory, pod_factory, request):
-            # change ceph full ratio to standard value
+            # Delete any clone pods that may have been created mid-test (RBD clones)
+            for clone_pod in self.clone_pods_list:
+                try:
+                    clone_pod.delete()
+                    clone_pod.ocp.wait_for_delete(
+                        resource_name=clone_pod.name, timeout=300
+                    )
+                except Exception:
+                    logger.warning(
+                        f"Failed to delete clone pod {clone_pod.name} during teardown"
+                    )
```

Ruff (0.15.21)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)